posix.mak

```diff
@@ -522,8 +522,8 @@ checkwhitespace: $(LIB) $(TOOLS_DIR)/checkwhitespace.d
 #############################
 
 ../dscanner:
-	git clone https://github.com/Hackerpilot/Dscanner ../dscanner
-	git -C ../dscanner checkout tags/v0.4.0
+	git clone https://github.com/dlang-community/Dscanner ../dscanner
+	git -C ../dscanner checkout phobos
```

Comment:
Yes, my hope is that with by using this feature at Phobos, the approval of the [respective PR] at Dscanner gets sped up. However, considering that in the past I was often blocked due to slow response times, having our own branch to add unapproved features (and workarounds) might not be a bad idea.